tools/manual_api_testing/push.rb

```diff
@@ -46,16 +56,25 @@
 
 # Submit the files in chunks of 20
 failures = []
-Parallel.each(files.each_slice(20)) do |slice|
+options = {}
+if sequential # Don't use parallel processing
+  options[:in_processes] = 0
+end
+slice_size = single ? 1 : 20
+Parallel.each(files.each_slice(slice_size), options) do |slice|
```

Comment:
Something I've been trying to do more of lately when writing Ruby scripts that depend on gems is to include a section before the requires; such as:

```
require 'bundler/inline'
gemfile do
  source 'https://rubygems.org'
  gem 'oauth2'
  gem 'activesupport'
  gem 'parallel'
end
```

Makes it a bit easier to run these sorts of scripts if the developer doesn't have these gems already installed.

Reply:
Nice, thanks!